sdk/openai/azure-ai-openai/src/samples/java/com/azure/ai/openai/ChatCompletionsWithYourData.java

```diff
@@ -39,40 +42,53 @@ public static void main(String[] args) {
             .credential(new AzureKeyCredential(azureOpenaiKey))
             .buildClient();
 
-        List<ChatRequestMessage> chatMessages = new ArrayList<>();
-        chatMessages.add(new ChatRequestUserMessage("How many of our customers are using the latest version of our SDK?"));
-
-        ChatCompletionsOptions chatCompletionsOptions = new ChatCompletionsOptions(chatMessages);
-
         // These configuration values come from your Azure Cognitive Search resource. Using Azure
         // Cognitive Search as a data source is briefly described in the "Azure OpenAI on
         // your data" quickstart, linked above. A more detailed guide can be found here:
         // https://learn.microsoft.com/azure/search/search-get-started-portal
-
         // Your Azure Cognitive Search endpoint, admin key, and index name
         String azureSearchEndpoint = "{azure-cognitive-search-endpoint}";
         String azureSearchAdminKey = "{azure-cognitive-search-key}";
-        String azureSearchIndexName = "{azure-cognitive-search-index-name}";
 
-        AzureCognitiveSearchChatExtensionParameters searchParameters = new AzureCognitiveSearchChatExtensionParameters(
-                azureSearchEndpoint,
-                azureSearchIndexName
-        );
-        searchParameters.setAuthentication(new OnYourDataApiKeyAuthenticationOptions(azureSearchAdminKey));
+        // The name of the index you want to use as a data source. This index name is created by running Azure Search Sample,
+        // https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/search/azure-search-documents/src/samples/java/com/azure/search/documents/VectorSearchExample.java#L75
+        String azureSearchIndexName = "{azure-cognitive-search-index-name}"; // "hotels-vector-sample-index"
```

Comment:
nit: Azure Cognitive Search rebranded as Azure AI Search, should update references with the new name